liiv2021a/openreview/content.tex

```diff
@@ -0,0 +1,339 @@
+    
+    \item \hypertarget{claim4}{\textbf{Dependency on ensemble size}}: Using 20 models in the ensemble does better than using 5 models, but there is no conclusive gain when using more than 20 models.
+    
+    \item \hypertarget{claim5}{\textbf{Dependency on temperature}}: It is necessary to use temperature of at least 5 to successfully distribution-distill the ensemble. Using higher initial temperatures do not result in conclusive improvement.    
+    
+    \item \hypertarget{claim5}{\textbf{Uncertainty decomposition}}: \EnDD \ trained with an auxiliary dataset is able to reconstruct the uncertainty decomposition made possible by ensembles.
+    
+\end{enumerate}
+
+We reproduce all experiments of the main article and most of the appendix, except for the use of CIFAR100 and Tiny Imagenet datasets. Some of these results can be found in our supplementary material. From their appendix, we do not reproduce Table 7 in appendix B. We did not recreate the OOD-detection plots when reproducing the ablation study.
+
+\section{Methodology}
+\label{sec:experiments-on-cifar-10}
+
+\subsection{Model description}
+We consider the same seven models as the original authors:
+\begin{itemize}
+    \item IND: A single classification model.
+    \item ENSM: An ensemble of independently trained IND models.
+    \item EnD: A single model distilling ENSM trained according to \cite{hinton2015distilling}. 
+    \item \EnDD: A single model distribution-distilling ENSM trained according to \cite{malinin2019ensemble}.   
+    \item EnD$_{\texttt{+AUX}}$: Like EnD, but trained with auxiliary data.
+    \item \EnDDaux: Like \EnDD, but trained with auxiliary data.
+    \item PN$_{\texttt{+AUX}}$: A PriorNet model with auxiliary data trained according to \cite{NIPS2018_7936}
+\end{itemize}
+
+These models are all based on almost identical VGG16 architectures \cite{VGG}, adapted to CIFAR-10 data as in \cite{NIPS2018_7936} by adding dropout, batch normalization and reducing the size of the fully connected layers. The only exception being that batch normalization is not used for PN. 
+
+\subsection{Dataset}
+The training set of CIFAR-10 was used as the primary training dataset. The training set of CIFAR-100 was used as an auxiliary dataset. For evaluating the classification task the test set of CIFAR-10 was used. For evaluating the out-of-distribution detection task the CIFAR-10 test set was used as in-domain dataset, while the LSUN test set was used as the out-of-domain dataset. Information about the datasets is listed in Table \ref{tab:datasets}.
+
+\begin{table}[]
+    \centering
+    \caption{Datasets used in the CIFAR-10 setting}
+    \addtolength{\leftskip} {-3cm}
+    \addtolength{\rightskip}{-3cm}
+    \begin{tabular}{r||c|c|c|c} 
+    \hline
+    Dataset & Samples & Classes & Dimensions & Link \\ [0.5ex] 
+    \hline
+    \hline
+    CIFAR-10 train & 50000 & 10 & 32x32x3 & \hrref{https://www.cs.toronto.edu/~kriz/cifar.html} \\
```

Comment:
Compilation complains here (undefined control sequence). It should be `href` right?